docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
     environment:
       MYSQL_DATABASE: ${DB_NAME}
       MYSQL_ROOT_PASSWORD: ${DB_PASS}
+      MYSQL_USER: ${DB_USER}
+      MYSQL_PASSWORD: ${DB_PASS}
```

Comment:
Fine to use same pass for root and non-root locally I think :shrug:

Reply:
Yeah, I agree!